presto-orc/src/main/java/io/prestosql/orc/StripeReader.java

```diff
@@ -232,7 +232,8 @@ private static boolean isSupportedStreamType(Stream stream, OrcTypeKind orcTypeK
     {
         if (stream.getStreamKind() == BLOOM_FILTER) {
             // non-utf8 bloom filters are not allowed for character types
-            return orcTypeKind != OrcTypeKind.STRING && orcTypeKind != OrcTypeKind.VARCHAR && orcTypeKind != OrcTypeKind.CHAR;
+            // non-utf8 bloom filters are not supported for timestamp
```

Comment:
Should this be "character types or timestamp"

Reply:
The line above says "non-utf8 bloom filters are not allowed for character types"